eng/source-build-release-official.yml

```diff
@@ -42,21 +62,18 @@ parameters:
   type: boolean
   default: true
 - name: createReleaseAnnouncement
-  displayName:  Create Release Announcement
+  displayName: Create Release Announcement
   type: boolean
   default: true
 - name: submitReleasePR
   displayName: Submit Release PR
   type: boolean
   default: true
-- name: useCustomTag
-  displayName: Use custom tag?
-  type: boolean
-  default: false
-- name: customTag
-  displayName: Custom release tag (e.g. v6.0.XYY-source-build)
+- name: createGitHubRelease
+  displayName: '[⚠️ 8.0] Create tag & release in dotnet/dotnet'
   type: string
-  default: ' '
+  values: [ 'skip', 'draft', 'full' ]
+  default: draft
```

Comment:
Why draft instead of full?  Would it make sense to have a `default` option that is dynamic in nature?  If DryRun is used, the default would be to skip, if not is would default to full.

Reply:
That's a good idea actually. I named it `auto`. I wonder if I should document the behaviour somewhere else than the code?